docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,18 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms`>> and model 
```

Comment:
In case people don't notice the formatting, it might be nice to qualify this:
```suggestion
If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms` aggs>> and model 
```